source/azure_iot.c

```diff
@@ -0,0 +1,455 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file azure_iot.c
+ *
+ * @brief Common Azure IoT code for FreeRTOS middleware.
+ *
+ */
+
+#include <stdio.h>
+
```

Comment:
still required?

Reply:
hmmm yea I don't know what this is here tbh

Reply:
oh wait it's memset

Reply:
replacing with `string.h`

Reply:
just to make sure that we are not causing any warning ... Please add this file to test cmake

https://github.com/Azure/azure-iot-middleware-freertos/blob/332117bbcbd4ef1592b3c275652d5c1e4335681f/tests/CMakeLists.txt#L28